src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,20 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+
+    @OneToMany
```

Comment:
The `@OneToMany` annotation should specify the mappedBy attribute to indicate the owner of the relationship. For example: `@OneToMany(mappedBy = "movie")` if the `Actor` class has a `movie` field.